--- origsrc/libidn-1.7/contrib/doxygen/Doxyfile.in	2008-02-19 10:53:35.000000000 -0600
+++ src/libidn-1.7/contrib/doxygen/Doxyfile.in	2008-04-13 16:21:57.812500000 -0500
@@ -459,7 +459,7 @@
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = ../../lib
+INPUT                  = @top_srcdir@/lib
 
 # If the value of the INPUT tag contains directories, you can use the 
 # FILE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
@@ -500,7 +500,7 @@
 # directories that contain example code fragments that are included (see 
 # the \include command).
 
-EXAMPLE_PATH           = ../../examples
+EXAMPLE_PATH           = @top_srcdir@/examples
 
 # If the value of the EXAMPLE_PATH tag contains directories, you can use the 
 # EXAMPLE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
@@ -530,7 +530,7 @@
 # to standard output.  If FILTER_PATTERNS is specified, this tag will be 
 # ignored.
 
-INPUT_FILTER           = ./gdoc2doxygen
+INPUT_FILTER           = @srcdir@/gdoc2doxygen
 
 # The FILTER_PATTERNS tag can be used to specify filters on a per file pattern 
 # basis.  Doxygen will compare the file name with each pattern and apply the 
